lms/static/scripts/frontend_apps/components/AppRoot.tsx

```diff
@@ -15,33 +17,105 @@ export type AppRootProps = {
   services: ServiceMap;
 };
 
+/**
+ * Return dummy configuration for the file picker app. This will be replaced
+ * with a call to the endpoint created in https://github.com/hypothesis/lms/pull/5120.
+ */
+/* istanbul ignore next */
+async function loadDummyFilePickerConfig(
+  config: ConfigObject
+): Promise<ConfigObject> {
+  // Add a fake delay so we can see the loading state initially.
+  await new Promise(resolve => setTimeout(resolve, 1000));
+
+  // Allow `simulateError` global to be set to trigger an error here.
+  if ((window as any).simulateError) {
```

Comment:
> I am guessing the cast to any here is because simulateError is not on the  window type? 

Yes. I didn't bother to augment the `window` global types because this is very much temporary code and I'm not planning to use this global anywhere else.